administrator/components/com_workflow/Model/TransitionsModel.php

```diff
@@ -64,7 +65,7 @@ public function __construct($config = array())
 	 *
 	 * @since  __DEPLOY_VERSION__
 	 */
-	protected function populateState($ordering = null, $direction = null)
+	protected function populateState($ordering = "t.ordering", $direction = "asc")
```

Comment:
Use ```'``` instead of ```"``` when possible! (also on all other positions)

Reply:
right, i still have that habit with double quote ...